client/explore-playlists/explore-playlists.html

```diff
@@ -1,3 +1,14 @@
-<div>
-  <h3>ALL playlist</h3>
+<div class="container" ng-controller="ExploreCtrl">
```

Comment:
remove <ng-controller="ExploreCtrl">, in app.js the ui.router assigned the controller
mentioned in the PR comment last time